src/objects/code-inl.h

```diff
@@ -171,7 +171,14 @@ Address Code::InstructionEnd(Isolate* isolate, Address pc) const {
 }
 
 int Code::GetOffsetFromInstructionStart(Isolate* isolate, Address pc) const {
+#if defined(__CHERI_PURE_CAPABILITY__)
```

Comment:
```suggestion
#if defined(__CHERI_PURE_CAPABILITY__) && defined(__aarch64__)
```

I think we should use this rather than just `__CHERI_PURE_CAPABILITY__` or `__CHERI_PURE_CAPABILITY__` paired with `V8_TARGET_ARCH_ARM64` because the latter would also activate when we are cross-compiling for Morello from a different CHERI architecture, which we don't actually want.

Reply:
Done.